manifests/4.4/cluster-logging.v4.4.0.clusterserviceversion.yaml

```diff
@@ -247,6 +247,8 @@ spec:
         - apiGroups:
           - rbac.authorization.k8s.io
           resources:
+          - roles
```

Comment:
I think this can be replaced by the static roles created by OLM.  See the manifest directory

Reply:
:+1:

Reply:
Moved to static roles